packages/cli/src/ui/contexts/KeypressContext.tsx

```diff
@@ -655,16 +655,21 @@ export function KeypressProvider({
     };
 
     let rl: readline.Interface;
+    const IME_ESCAPE_CODE_TIMEOUT_MS = 20;
```

Comment:
adding a constant for this is reasonable. Please add this to the constants file and add a comment why 20ms is recommended. If possible please also add a test that will fail if the escape code timeout is moved back to zero by accident. Ideal test: simulate sending escape codes for an IME language with suitable delays so they are not processed correctly without this.